src/main/proto/cosmas.proto

```diff
@@ -64,6 +65,16 @@ message CommitVersionResponse {
 
 }
 
+message FileInformation {
+    message Patch {

```

Comment:
any sense to extract this Patch message to top-level and use it both here and in `CreatePatchRequest`?

Reply:
ok